src/azure-cli/azure/cli/command_modules/appservice/_help.py

```diff
@@ -1542,3 +1542,145 @@
+"""
+
+helps['webapp hybrid-connection list'] = """
+type: command
+short sumamry: list the hybrid-connections on a webapp
+examples:
+  - name: list the hybrid-connections on a webapp
+    text: az webapp hybrid-connection list -g MyResourceGroup -n MyWebapp -s [slot]
+"""
+
+helps['webapp hybrid-connection add'] = """
+type: command
+short sumamry: add a hybrid-connection to a webapp
+examples:
+  - name: add a hybrid-connection to a webapp
+    text: az webapp hybrid-connection add -g MyResourceGroup -n MyWebapp --namespace [HybridConnectionNamespace] --hybrid-connection [HybridConnectionName] -s [slot]
+"""
+
+helps['webapp hybrid-connection remove'] = """
+type: command
+short sumamry: remove a hybrid-connection from a webapp
+examples:
+  - name: remove a hybrid-connection from a webapp
+    text: az webapp hybrid-connection remove  -g MyResourceGroup -n MyWebapp --namespace [HybridConnectionNamespace] --hybrid-connection [HybridConnectionName] -s [slot]
+"""
+
+helps['functionapp hybrid-connection'] = """
+type: group
+short summary: methods that list, add and remove hybrid-connections from functionapp
+"""
+
+helps['functionapp hybrid-connection list'] = """
+type: command
+short sumamry: list the hybrid-connections on a functionapp
+examples:
+  - name: list the hybrid-connections on a functionapp
+    text: az functionapp hybrid-connection list -g MyResourceGroup -n MyWebapp -s [slot]
+"""
+
+helps['functionapp hybrid-connection add'] = """
+type: command
+short sumamry: add a hybrid-connection to a functionapp
+examples:
+  - name: add a hybrid-connection to a functionapp
+    text: az functionapp hybrid-connection add -g MyResourceGroup -n MyWebapp --namespace [HybridConnectionNamespace] --hybrid-connection [HybridConnectionName] -s [slot]
+"""
+
+helps['functionapp hybrid-connection remove'] = """
+type: command
+short sumamry: remove a hybrid-connection from a functionapp
+examples:
+  - name: remove a hybrid-connection from a functionapp
+    text: az functionapp hybrid-connection remove -g MyResourceGroup -n MyWebapp --namespace [HybridConnectionNamespace] --hybrid-connection [HybridConnectionName] -s [slot]
+"""
+
+helps['appservice hybrid-connection'] = """
+type: group
+short summary: set the key that all apps in an appservice plan use to connect to the hybrid-connections in that appservice plan
+"""
+
+helps['appservice hybrid-connection set-key'] = """
+type: command
+short summary: set the key that all apps in an appservice plan use to connect to the hybrid-connections in that appservice plan
+examples:
+  - name:  set the key that all apps in an appservice plan use to connect to the hybrid-connections in that appservice plan
+    text: az appservice hybrid-connection set-key -g MyResourceGroup --plan MyAppServicePlan --namespace [HybridConectionNamespace] --hybrid-connection [HybridConnectionName] --key-type ["primary"/"secondary"]
+"""
+
+helps['appservice plan vnet-integration'] = """
```

Comment:
Also inoder to avoid confusion i would move these to the appservice help section above. Noticed that none of the others tests use [] can you update to not use these  as well?